mooncake-transfer-engine/include/transfer_engine_c.h

```diff
@@ -126,13 +133,14 @@ int unregisterLocalMemoryBatch(transfer_engine_t engine, void **addr_list,
 
 batch_id_t allocateBatchID(transfer_engine_t engine, size_t batch_size);
 
-int submitTransfer(transfer_engine_t engine, batch_id_t batch_id,
-                   struct transfer_request *entries, size_t count);
+status_t submitTransfer(transfer_engine_t engine, batch_id_t batch_id,
```

Comment:
keep int would be better?

Reply:
@doujiang24 I prefer to change to status_t, in the new PRs, I plan to change all the external interface return value into status_t, in which it's able to expose more information for upper layer to debug problems.
In the mean-time, change all the internal function return values into Status. Because in the future, we may implement SSD offload, the return status can hold more information, such as NVMe error code, disk id, etc.